migrations/20190821201450-session.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  // This is from https://github.com/voxpelli/node-connect-pg-simple/blob/HEAD/table.sql
+  // It is a requirement to use the Postgres store
+  return db.runSql(
+    `CREATE TABLE "session" (
+      "sid" varchar NOT NULL COLLATE "default",
+      "sess" json NOT NULL,
```

Comment:
Nit: Do you mind if we use the full word (I believe it's "session"?) ? I find it more readable. Happy to not do that though, if there's a space constraint or something.

Reply:
The constraint is the library `connect-pg-simple` - see the part about `table.sql`

[https://www.npmjs.com/package/connect-pg-simple#installation]()

Reply:
Ah okay, fair enough